app/models/offer.rb

```diff
@@ -1,4 +1,15 @@
 class Offer < ApplicationRecord
   belongs_to :user
   has_many :bookings
+  has_one_attached :photo
+
+  CATEGORY = ['German', 'Italian', 'Vietnamese', 'French', 'Indian', 'Fusion'].freeze
+
+  # An offer must have a unique title and a description
+  validates :title, uniqueness: true, presence: true
+  validates :description, presence: true
+
+  validates :price, numericality: { only_integer: true }, presence: true
```

Comment:
the price is float. Not sure what only_integer does though..

Reply:
You're right. I need to check this again